### DISCUSSION TITLE
About This script

### DISCUSSION THREAD

**Discussion by Maverick1000** [LocuraDU/DU-Locura-Storage-Info#4] [General]
I have a factory the was created by a dual univers player and one of the screen shows the following 
![image](https://github.com/user-attachments/assets/b7c332f5-bf43-4c64-bfc5-dce9a74c7554)
, but I notice the latest of this mode does not do that any more, did you change it. If so, do stil have a version the does what the image showes?

**Comment by CredenceHamby** [comment]
The latest version of the factory or this script?
